docs/commands.md

```
@@ -64,28 +67,124 @@ The interaction handler is shared with `/dl` (see `src/bot/interactionCreate.ts`
+| `name` | `STRING` (3) | yes | `Thread Name` | Thread title. Truncated to 80 characters before being round-tripped via the Modal `customId`; the same truncated value is used as the actual thread name. |
+
+> URLs are **not** a slash-command option. They are read from the Modal input that opens as the immediate response to the slash command.
+
+### Modal input
+
+When `/threaddl` is invoked, the bot's first response is **not** a deferred follow-up but a Modal:
+
+| Modal field | Value |
+| --- | --- |
+| `title` | `Add URLs to "<threadName-truncated-to-40>"` |
+| `customId` | `threaddl|<threadName-truncated-to-80>` |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 2; Actual: 3; Too many cells, extra data will be missing

(MD056, table-column-count)